third_party/googletest/webport/build/Makefile

```diff
@@ -78,14 +78,24 @@ endif
 #   the compilation according to prepared scripts.
 #
 # Explanation of parameters to cmake:
-# B: build directory.
+# E env: Wraps the further second cmake call into the "env" tool that allows to
```

Comment:
nit: Isn't it the first cmake call, rather than the second? Also, delete "further".

Reply:
The first `cmake E env ...` call is used to set up the environment values for the second `cmake ../../src` one. If the comment doesn't do a good job in explaining that, I'm happy to rephrase it.

Reply:
I see. I think I would go with `Wraps the succeeding cmake call`